server/data/accreditedProgrammesApi/referralClient.test.ts

```diff
@@ -120,20 +121,25 @@ pactWith({ consumer: 'Accredited Programmes UI', provider: 'Accredited Programme
 
     describe('with query parameters', () => {
       const paginatedReferralSummaries: Paginated<ReferralSummary> = {
-        content: [referralSummaryFactory.build({ status: 'referral_submitted' })],
+        content: FactoryHelpers.buildListWith(
+          referralSummaryFactory,
+          { audiences: ['General offence'], status: 'referral_started' },
+          16,
+        ),
         pageIsEmpty: false,
-        pageNumber: 0,
-        pageSize: 10,
-        totalElements: 1,
-        totalPages: 1,
+        pageNumber: 1,
+        pageSize: 15,
+        totalElements: 16,
+        totalPages: 2,
       }
 
       beforeEach(() => {
         provider.addInteraction({
+          // DOES THIS NOT CONFLICT WITH THE STATE ABOVE? 1 vs 16 with specific params
```

Comment:
I'm not sure, but we can test this one out locally against the API and see how it behaves